packages/terra-action-header/tests/jest/ActionHeader.test.jsx

```diff
@@ -63,31 +63,31 @@ describe('ActionHeader', () => {
   });
 
   it('should render an action header with maximize button and title', () => {
-    const actionHeader = <ActionHeader title="Action Header" onMaximize={() => {}} />;
+    const actionHeader = <ActionHeader level={1} title="Action Header" onMaximize={() => {}} />;
     const wrapper = shallowWithIntl(actionHeader).dive();
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render an action header with minimize button and title', () => {
-    const actionHeader = <ActionHeader title="Action Header" onMinimize={() => {}} />;
+    const actionHeader = <ActionHeader level={1} title="Action Header" onMinimize={() => {}} />;
     const wrapper = shallowWithIntl(actionHeader).dive();
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render an action header with next and previous buttons and title', () => {
-    const actionHeader = <ActionHeader title="Action Header" onNext={() => {}} onPrevious={() => {}} />;
+    const actionHeader = <ActionHeader level={1} title="Action Header" onNext={() => {}} onPrevious={() => {}} />;
     const wrapper = shallowWithIntl(actionHeader).dive();
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render an action header with title, enabled next button, and disabled previous button', () => {
-    const actionHeader = <ActionHeader title="Action Header" onNext={() => {}} />;
+    const actionHeader = <ActionHeader level={1} title="Action Header" onNext={() => {}} />;
```

Comment:
do we have test for not providing value for `level` prop.

Reply:
There is a wdio [test](https://github.com/cerner/terra-core/blob/main/packages/terra-core-docs/src/terra-dev-site/test/action-header/EndElementActionHeader.test.jsx) with no text and level.